docs/visual-testing/integrations/java.md

```diff
@@ -0,0 +1,172 @@
+---
+sidebar_label: Java
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Java WebDriver Integration
+
+## Introduction
+
+This guide requires an existing Java JUnit / TestNG setup.<br />
+You can alternatively take a look to our [example repository](#examples).
+
+Sauce Labs Visual provides an library allowing integration with [WebDriver](https://www.selenium.dev/documentation/webdriver/).
+
+Sauce Labs Visual plugin provides a library exposing a `VisualApi` object that provides actions:
+- `visual.check()`: Takes a screenshot and send it to Sauce Labs Visual for comparison.
+- `visual.checkResults()`: Waits for diff calculations to complete and returns a summary of results.
+
+Sample usage:
+```java
+    assertEquals(visual.checkResults().get(DiffStatus.UNAPPROVED), 2);
```

Comment:
@kb-kerem 
Where do ew explain what the object looks like?

Reply:
Pushed a change to explain it in Advanced usage section. Also provided a link to there.
@konraddysput now it's using a constant